### PR TITLE
Fix: checking node type before making node active for specific DDC action

### PR DESCRIPTION
**Pull request by yahortsaryk**

### CODE REVIEW

File: pallets/ddc-staking/src/lib.rs [hunk 1]
```diff
@@ -173,14 +173,14 @@ pub mod pallet {
 	pub type Ledger<T: Config> =
 		StorageMap<_, Blake2_128Concat, T::AccountId, StakingLedger<T::AccountId, BalanceOf<T>, T>>;
 
-	/// The map of (wannabe) CDN participants stash keys to the DDC cluster ID they wish to
+	/// The map of (wannable) CDN participants stash keys to the DDC cluster ID they wish to
```
Comment (khssnv) [inline pallets/ddc-staking/src/lib.rs:176]: The comment from the FRAME stacking pallet, could be rephrased.

File: pallets/ddc-staking/src/lib.rs [hunk 2]
```diff
@@ -173,14 +173,14 @@ pub mod pallet {
...
 	/// participate into.
 	#[pallet::storage]
 	#[pallet::getter(fn cdns)]
 	pub type CDNs<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, ClusterId>;
 
-	/// The map of (wannabe) storage network participants stash keys to the DDC cluster ID they wish
-	/// to participate into.
+	/// The map of (wannable) storage network participants stash keys to the DDC cluster ID they
```
Comment (khssnv) [inline pallets/ddc-staking/src/lib.rs:182]: The comment from the FRAME stacking pallet, could be rephrased.

**Review by khssnv — DISMISSED** [review]

File: pallets/ddc-staking/src/lib.rs [hunk 3]
```diff
@@ -173,14 +173,14 @@ pub mod pallet {
...
+	/// wish to participate into.
```
Comment (rakanalh) [inline pallets/ddc-staking/src/lib.rs:183]: Can you provide a better doc string here? The storage is called `Storages` but what is it a storage for?

File: runtime/cere-dev/src/lib.rs [hunk 4]
```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48013,
+	spec_version: 48014,
```
Comment (rakanalh) [inline runtime/cere-dev/src/lib.rs:132]: How about we don't increment spec_version on every PR but rather do that on releases?

File: pallets/ddc-staking/src/lib.rs (hunk 2 above)
Comment (yahortsaryk) [reply pallets/ddc-staking/src/lib.rs]: Agree, I didn't know there is "wannabe" word in English, so I thought it's a typo 😅

File: pallets/ddc-staking/src/lib.rs (hunk 1 above)
Comment (yahortsaryk) [reply pallets/ddc-staking/src/lib.rs]: Agree, I didn't know there is "wannabe" word in English, so I thought it's a typo 😅

File: pallets/ddc-staking/src/lib.rs (hunk 3 above)
Comment (yahortsaryk) [reply pallets/ddc-staking/src/lib.rs:183]: Yes, they are Storage nodes that participate in DDC network

File: runtime/cere-dev/src/lib.rs (hunk 4 above)
Comment (yahortsaryk) [reply runtime/cere-dev/src/lib.rs:132]: I propose to keep it as it is currently until we don't have a well-established and clear process on how we update Devnet runtime and runtime for the environments. I'm not sure what is the best way to deliver this fix to Devnet ASAP.

**Review by rakanalh — APPROVED** [review]

**Pull request merged**